### PR TITLE
Migrate to swift 5

### PR DESCRIPTION
**Pull request by niil-ohlin**
Recursive signals are failing when using UIKit kvo signals. in Xcode 10.2

### CODE REVIEW

File: FlowTests/SignalConcurrenceyTests.swift [hunk 1]
```diff
@@ -349,4 +354,30 @@ class SignalConcurrenceyTests: XCTestCase {
...
     }
 
+    class TestObject: NSObject {
+        @objc var isSelected: Bool = false
+    }
+
+    func testKVO() {
+        #if canImport(UIKit)
+
+        let object = UIButton()
```
Comment (niil-ohlin) [inline FlowTests/SignalConcurrenceyTests.swift]: The test compiles and runs when you instantiate it with `TestObject` instead of `UIButton`.

**Comment by TBernatskaya** [comment]
I'll leave some useful links here:
[Swift 5 Release Notes for Xcode 10.2](https://developer.apple.com/documentation/xcode_release_notes/xcode_10_2_release_notes/swift_5_release_notes_for_xcode_10_2)
[Swift 5 Exclusivity Enforcement](https://swift.org/blog/swift-5-exclusivity/)

**Comment by niil-ohlin** [comment]
@TBernatskaya solved it!

**Comment by niil-ohlin** [comment]
Form builds and tests are passing with this version of Flow.

**Comment by TBernatskaya** [comment]
> @TBernatskaya solved it!

Great job 👏

**Pull request merged**